src/tests/PhysicsTest.cpp

```diff
@@ -26,44 +26,63 @@ const std::string physicsConfigFile =
     Cr::Utility::Directory::join(SCENE_DATASETS,
                                  "../default.phys_scene_config.json");
 
-TEST(PhysicsTest, JoinCompound) {
-  LOG(INFO) << "Starting physics test";
-  // must create a context and renderer first
-  esp::gfx::WindowlessContext::uptr context_ =
-      esp::gfx::WindowlessContext::create_unique(0);
-  std::shared_ptr<esp::gfx::Renderer> renderer_ = esp::gfx::Renderer::create();
+class PhysicsManagerTest : public testing::Test {
+ protected:
+  void SetUp() override {
+    context_ = esp::gfx::WindowlessContext::create_unique(0);
+    renderer_ = esp::gfx::Renderer::create();
+
+    sceneID_ = sceneManager_.initSceneGraph();
+  };
+
+  void initScene(const std::string sceneFile) {
+    sceneFile_ = sceneFile;
+
+    const esp::assets::AssetInfo info =
+        esp::assets::AssetInfo::fromPath(sceneFile);
+
+    auto& sceneGraph = sceneManager_.getSceneGraph(sceneID_);
+    esp::scene::SceneNode* navSceneNode =
+        &sceneGraph.getRootNode().createChild();
+    auto& drawables = sceneManager_.getSceneGraph(sceneID_).getDrawables();
+    resourceManager_.loadScene(info, physicsManager_, navSceneNode, &drawables,
+                               physicsConfigFile);
+  }
 
   // must declare these in this order due to avoid deallocation errors
-  ResourceManager resourceManager;
+  esp::gfx::WindowlessContext::uptr context_;
+  esp::gfx::Renderer::ptr renderer_;
+
+  ResourceManager resourceManager_;
   SceneManager sceneManager_;
-  std::shared_ptr<PhysicsManager> physicsManager_;
+  PhysicsManager::ptr physicsManager_;
+
+  std::string sceneFile_;
```

Comment:
Why is this needed? Its not used anywhere.

Reply:
👍 Was used, but no longer necessary. I'll add back later if needed.